core/tests/protractor_utils/StoryEditorPage.js

```diff
@@ -442,17 +474,24 @@ var StoryEditorPage = function() {
     await waitFor.elementToBeClickable(
       addPrerequisiteSkillButton,
       'addPrerequisitesSkillButton takes too long to be clickable');
-    await addPrerequisiteSkillButton.click();
+    await action.click(
+      'Add Prerequisite Skill Button', addPrerequisiteSkillButton);
     var selectSkillModal = await this.getSelectSkillModal();
     await selectSkillModal.selectSkill(skillName);
   };
 
   this.deleteAcquiredSkillByIndex = async function(index) {
-    await deleteAcquiredSkillButton.get(index).click();
+    await action.click(
+      'Delete Acquired Skill Button',
+      deleteAcquiredSkillButton.get(index)
+    );
   };
 
   this.deletePrerequisiteSkillByIndex = async function(index) {
-    await deletePrerequisiteSkillButton.get(index).click();
+    await action.click(
+      'Delete PrerequisiteSkill Button',
```

Comment:
ditto

Reply:
Thank you @U8NWXD for your reviews. I'll make the require changes.

Reply:
@U8NWXD I've made the changes but am not able to push changes as it is showing some errors in `oppia_tools/esprima-4.0.1/esprima/parser.py` file.